packages/n4s/docs/rules.md

```diff
@@ -1280,3 +1282,52 @@ enforce('10.12').isPositive(); //passes
 enforce(-10).isPositive(); // throws
 enforce('-10.12').isPositive(); // throws
 ```
+
+## isKeyOf
```

Comment:
This is good, but just a request - I am in the process of replacing the docs folder and moving it someplace else. Can you please move this change to here: https://github.com/ealush/vest/blob/latest/website/docs/enforce/enforce_rules.md instead?

Reply:
Sure no problem. But you gave me the same path I pushed the code on, do you mean another path?